requirements/tracker

```diff
@@ -0,0 +1,7 @@
+# Please make sure to pin all dependency versions, in order to avoid unwanted
+# functional and integration breaks caused by external code updates.
+#
+# These are web-api tracking dependencies, so pin them.
```

Comment:
I'm not sure I understand why this is pinned. I'd prefer that we can deploy an upgrade to this code without requiring a Core upgrade.

Reply:
Agreed, I guess pinning only makes sense for dev deps.